src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -126,24 +132,29 @@ public static void validateDictionaries( final String name1,
                 // reference AND when the user's intervals actually include one or more of the contigs that are
                 // indexed differently from the reference. In this case, the engine will fail to correctly serve
                 // up the reads from those contigs, so throw an exception.
-                if ( isReadsToReferenceComparison && intervals != null ) {
-
-                    final Set<String> misindexedContigs = findMisindexedContigsInIntervals(intervals, dict1, dict2);
-
-                    if ( ! misindexedContigs.isEmpty() ) {
-                        final String msg = "The following contigs included in the intervals to process have " +
-                                        "different indices in the sequence dictionaries for the reads vs. " +
-                                        "the reference: %s.  As a result, the GATK engine will not correctly " +
-                                        "process reads from these contigs. You should either fix the sequence " +
-                                        "dictionaries for your reads so that these contigs have the same indices " +
-                                        "as in the sequence dictionary for your reference, or exclude these contigs " +
-                                        "from your intervals.";
-                        final UserException ex = new UserException.IncompatibleSequenceDictionaries(msg, name1, dict1, name2, dict2);
-                        throw ex;
+                if ( isReadsToReferenceComparison ) {
+                    if (intervals != null) {
+
+                        final Set<String> misindexedContigs = findMisindexedContigsInIntervals(intervals, dict1, dict2);
+
+                        if (!misindexedContigs.isEmpty()) {
+                            final String msg = "The following contigs included in the intervals to process have " +
+                                    "different indices in the sequence dictionaries for the reads vs. " +
+                                    "the reference: %s.  As a result, the GATK engine will not correctly " +
+                                    "process reads from these contigs. You should either fix the sequence " +
+                                    "dictionaries for your reads so that these contigs have the same indices " +
+                                    "as in the sequence dictionary for your reference, or exclude these contigs " +
+                                    "from your intervals.";
+                            final UserException ex = new UserException.IncompatibleSequenceDictionaries(msg, name1, dict1, name2, dict2);
+                            throw ex;
+                        }
+                    } else {
+                        throw new UserException.IncompatibleSequenceDictionaries("Reference is missing contigs found in reads", name1, dict1, name2, dict2);
```

Comment:
Actually, let's change this block of code so that we always throw if we get `DIFFERENT_INDICES` and `isReadsToReferenceComparison` is true (if it proves too annoying/restrictive, we can relax it later). This also means that you can remove the intervals argument to this method.